Rover/version.h

```diff
@@ -15,3 +15,5 @@
 #define FW_MINOR 1
 #define FW_PATCH 0
 #define FW_TYPE FIRMWARE_VERSION_TYPE_DEV
+
+#include <AP_Common/AP_FWVersionDefine.h>
```

Comment:
this brings in the structure declaration which seems wrong for the .h file

Reply:
my concern is the fwver symbol being defined twice

Reply:
In theory that's what FORCE_VERSION_H_INCLUDE is protecting against. So however includes that file need to define FORCE_VERSION_H_INCLUDE to get the symbol defined. And in theory there is only one place in the vehicle code doing that